alert-common/src/main/java/com/synopsys/integration/alert/common/event/ProviderCallbackEvent.java

```diff
@@ -8,9 +8,11 @@
 public class ProviderCallbackEvent extends AlertEvent {
     private final String callbackUrl;
     private final String notificationType;
+
+    // TODO consider splitting channelDestination out into something resembling "key" and "link"
     private final LinkableItem channelDestination;
     private final IssueOperation operation;
-    private final String channelEventDescription;
+    private final String channelActionSummary;
```

Comment:
I would love some variable name suggestions here.

- `channelDestination` represents the location that the channel posted to (e.g. name="IssueKey", Value="JIRA-123", url= "https://jira/browse/JIRA-123").
- `operation` is currently tightly coupled to `IssueOperation`, but we might want to change that.
- `channelActionSummary` represents a summary of the channel message (e.g. the Jira issue summary).